src/utils/isNodeProcess.ts

```diff
@@ -0,0 +1,3 @@
+export const isNode =
+  Object.prototype.toString.call(typeof process !== 'undefind') ===
```

Comment:
Please, could you make a function out of it? That way figuring out if it's Node is an _action_, and it's bound to the context where you call it, as opposed to where you declare it. Thanks!